src/cascadia/TerminalApp/defaults.json

```diff
@@ -234,6 +234,7 @@
         { "command": { "action": "switchToTab", "index": 8 }, "keys": ["ctrl+alt+9"] },
         { "command": "decreaseFontSize", "keys": ["ctrl+-"] },
         { "command": "increaseFontSize", "keys": ["ctrl+="] },
+        { "command": "toggleFind", "keys": [ "ctrl+f" ]},

```

Comment:
Are we concerned about stealing Ctrl+F by default? Should it be Ctrl+Shift+F? This reflects back to the Copy/Paste discussion. Probably consult with @cinnamon-msft, @zadjii-msft, and @DHowett-MSFT on it.

Reply:
Did the discussion of this happen somewhere I can't see, or did you just change it?

Reply:
I just changed it, Griese also requested this change.